utils/net/iputils_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package net
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestAssemblyIPList(t *testing.T) {
+	tests := []struct {
+		name    string
+		ipStr   string
+		wantErr bool
+	}{
+		{
+			name:    "baseData1",
+			ipStr:   "10.110.101.1-10.110.101.5",
+			wantErr: false,
+		},
+		/*{
```

Comment:
I am not sure why it would fail if I add this case.

Reply:
> I am not sure why it would fail if I add this case.

Maybe because 0.x.x.x is not a real ip address